presentations/en/Strategy-Continuous-Learning-Automating-Improving.md

```diff
@@ -93,23 +94,32 @@ FY 2019-2020 Q4
 
 - [Unblock access to specific SaaS for IITB employees](web-services-access.html)
 - Invest in bandwidth, Data Loss Protection (DLP) and a document classification tool to enable more open access to Internet
-- Add all approved software to Application Catalogue
+- Automate the addition all approved software to Application Catalogue
 
 ---
 
 ## IT Strategy Will
 
 - Organize a Hackathon
-- Coordonnate communications of changes
+- Coordonate communications of changes
+- Assist with, promote, and support the creation of a DevOps Dojo
+- Assist with, promote, and support the creation of Champions
```

Comment:
This sentence is a bit odd to me. I think another word than "creation" should be used for the Champions part.